cmd/createenr.go

```diff
@@ -61,22 +65,47 @@ func runCreateEnrCmd(w io.Writer, config p2p.Config, dataDir string) error {
 		return err
 	}
 
-	localEnode, db, err := p2p.NewLocalEnode(config, key)
+	keyPath := p2p.KeyPath(dataDir)
+
+	enrStr, err := createENR(key, config)
 	if err != nil {
-		return errors.Wrap(err, "failed to open enode")
+		return err
 	}
-	defer db.Close()
-
-	keyPath := p2p.KeyPath(dataDir)
 
 	_, _ = fmt.Fprintf(w, "Created ENR private key: %s\n", keyPath)
-	_, _ = fmt.Fprintln(w, localEnode.Node().String())
+	_, _ = fmt.Fprintln(w, enrStr)
 
 	writeEnrWarning(w, keyPath)
 
 	return nil
 }
 
+func createENR(key *ecdsa.PrivateKey, config p2p.Config) (string, error) {
+	tcpAddrs, err := config.ParseTCPAddrs()
+	if err != nil {
+		return "", err
+	}
+
+	var r enr.Record
+	for _, addr := range tcpAddrs {
+		r.Set(enr.IP(addr.IP))
```

Comment:
we should not have two different ways of calculating ENRs from config. for example, you are not including `ExternalIP` or `ExternalHost`. We need to think a bit more about what we encode into ENRs and how they are used.